modules/efk-logging-clo-status.adoc

```diff
@@ -0,0 +1,255 @@
+<5> Information on the Kibana pods.
+
+
+[id="efk-logging-clo-status-message_{context}"]
+== Example condition messages
+
+The following are examples of some condition messages from the `Status.Nodes` section of the cluster logging instance.
+
+
+// https://github.com/openshift/elasticsearch-operator/pull/92
+
+This status message indicates a node has exceeded the configured low watermark and no shard will be allocated to this node.
+
+----
+  nodes:
+  - conditions:
+    - lastTransitionTime: 2019-03-15T15:57:22Z
+      message: Disk storage usage for node is 27.5gb (36.74%). Shards will be not
+        be allocated on this node.
+      reason: Disk Watermark Low
+      status: "True"
+      type: NodeStorage
+    deploymentName: example-elasticsearch-clientdatamaster-0-1
+    upgradeStatus: {}
+----
+
+This status message indicates a node has exceeded the configured high watermark and shard will be relocated to other nodes.
+
+----
+  nodes:
+  - conditions:
+    - lastTransitionTime: 2019-03-15T16:04:45Z
+      message: Disk storage usage for node is 27.5gb (36.74%). Shards will be relocated
+        from this node.
+      reason: Disk Watermark High
+      status: "True"
+      type: NodeStorage
+    deploymentName: cluster-logging-operator
+    upgradeStatus: {}
+----
+
+This status message shows the Elasticsearch node selector in the CR does not match any nodes in the cluster:
+
+----
+    Elasticsearch Status:
+      Shard Allocation Enabled:  shard allocation unknown
+      Cluster:
+        Active Primary Shards:  0
+        Active Shards:          0
+        Initializing Shards:    0
+        Num Data Nodes:         0
+        Num Nodes:              0
+        Pending Tasks:          0
+        Relocating Shards:      0
+        Status:                 cluster health unknown
+        Unassigned Shards:      0
+      Cluster Name:             elasticsearch
+      Node Conditions:
+        elasticsearch-cdm-mkkdys93-1:
+          Last Transition Time:  2019-06-26T03:37:32Z
+          Message:               0/5 nodes are available: 5 node(s) didn't match node selector.
+          Reason:                Unschedulable
+          Status:                True
+          Type:                  Unschedulable
+        elasticsearch-cdm-mkkdys93-2:
+      Node Count:  2
+      Pods:
+        Client:
+          Failed:
+          Not Ready:
+            elasticsearch-cdm-mkkdys93-1-75dd69dccd-f7f49
+            elasticsearch-cdm-mkkdys93-2-67c64f5f4c-n58vl
+          Ready:
+        Data:
+          Failed:
+          Not Ready:
+            elasticsearch-cdm-mkkdys93-1-75dd69dccd-f7f49
+            elasticsearch-cdm-mkkdys93-2-67c64f5f4c-n58vl
+          Ready:
+        Master:
+          Failed:
+          Not Ready:
+            elasticsearch-cdm-mkkdys93-1-75dd69dccd-f7f49
+            elasticsearch-cdm-mkkdys93-2-67c64f5f4c-n58vl
+          Ready:
+----
+
+This status message indicates that the Elasticsearch CR uses a non-existent PVC.
```

Comment:
Could we use some other words to describe the condition? The root cause here is the PVC could not bind to PV.